<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarWidget</class>
 <widget class="QWidget" name="CalendarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="Calendar_btn_YSubtract">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{ 
	background-color:rgb(64,64,64);
	font-size:28px; 
	border-radius:8px;
	color:rgb(255,255,255);
}
QPushButton:pressed
{
	background-color:rgb(31, 78, 121);
	color:rgb(255,255,255);
	font-size:28px;
	border-radius:8px;
	/*border: 2px solid #1B72BF;    */
}
</string>
       </property>
       <property name="text">
        <string notr="true">&lt;&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Calendar_btn_MSubtract">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{ 
	background-color:rgb(64,64,64);
	font-size:28px; 
	border-radius:8px;
	color:rgb(255,255,255);
}
QPushButton:pressed
{
	background-color:rgb(31, 78, 121);
	color:rgb(255,255,255);
	font-size:28px;
	border-radius:8px;
	/*border: 2px solid #1B72BF;    */
}
</string>
       </property>
       <property name="text">
        <string notr="true">&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Calendar_lab_Now">
       <property name="styleSheet">
        <string notr="true">QLabel
{
	color: rgb(63, 63, 63);
	background-color: transparent;
	font-size:28px;
}</string>
       </property>
       <property name="text">
        <string notr="true">2021-07</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Calendar_btn_MAdd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{ 
	background-color:rgb(64,64,64);
	font-size:28px; 
	border-radius:8px;
	color:rgb(255,255,255);
}
QPushButton:pressed
{
	background-color:rgb(31, 78, 121);
	color:rgb(255,255,255);
	font-size:28px;
	border-radius:8px;
	/*border: 2px solid #1B72BF;    */
}
</string>
       </property>
       <property name="text">
        <string notr="true">&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Calendar_btn_YAdd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{ 
	background-color:rgb(64,64,64);
	font-size:28px; 
	border-radius:8px;
	color:rgb(255,255,255);
}
QPushButton:pressed
{
	background-color:rgb(31, 78, 121);
	color:rgb(255,255,255);
	font-size:28px;
	border-radius:8px;
	/*border: 2px solid #1B72BF;    */
}
</string>
       </property>
       <property name="text">
        <string notr="true">&gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Calendar_layout_btn" stretch="1,1,1">
     <item>
      <widget class="QPushButton" name="Calendar_btn_subtract">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{ 
	background-color:rgb(64,64,64);
	font-size:56px; 
	border-radius:8px;
	color:rgb(255,255,255);
}
QPushButton:pressed
{
	background-color:rgb(31, 78, 121);
	color:rgb(255,255,255);
	font-size:56px;
	border-radius:8px;
	/*border: 2px solid #1B72BF;    */
}
</string>
       </property>
       <property name="text">
        <string notr="true">-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Calendar_Edit_num">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
	font-size:56px;
	color:rgb(31, 78, 121);
	background:white;
	border-style:outset;
	border-radius:8px;
}
</string>
       </property>
       <property name="text">
        <string notr="true">11</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Calendar_btn_add">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{ 
	background-color:rgb(64,64,64);
	font-size:56px; 
	border-radius:8px;
	color:rgb(255,255,255);
}
QPushButton:pressed
{
	background-color:rgb(31, 78, 121);
	color:rgb(255,255,255);
	font-size:56px;
	border-radius:8px;
	/*border: 2px solid #1B72BF;    */
}
</string>
       </property>
       <property name="text">
        <string notr="true">+</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="Calendar_lab_ntime">
       <property name="styleSheet">
        <string notr="true">QLabel
{
	color: rgb(63, 63, 63);
	background-color: transparent;
	font-size:28px;
}</string>
       </property>
       <property name="text">
        <string notr="true">2021-07-21</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Calendar_btn_Hour">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{ 
	background-color:rgb(255,255,255);
	font-size:28px; 
	border-radius:8px;
	color:rgb(31, 78, 121);
	border: 2px solid #1f4e79;    
}
QPushButton:hover
{
	background-color:rgb(31, 78, 121);
	font-size:28px;
	border-radius:8px;
	color:white;
	border: 0px solid #1f4e79;    
}

QPushButton:checked{
	background-color:rgb(31, 78, 121);
	font-size:28px;
	border-radius:8px;
	color:white;
	border: 0px solid #1f4e79;    
};

</string>
       </property>
       <property name="text">
        <string notr="true">07</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	color: rgb(137, 137, 137);
	background-color: transparent;
	font-size:28px;
}</string>
       </property>
       <property name="text">
        <string notr="true">:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Calendar_btn_Minute">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{ 
	background-color:rgb(255,255,255);
	font-size:28px; 
	border-radius:8px;
	color:rgb(31, 78, 121);
	border: 2px solid #1f4e79;    
}
QPushButton:hover
{
	background-color:rgb(31, 78, 121);
	font-size:28px;
	border-radius:8px;
	color:white;
	border: 0px solid #1f4e79;    
}

QPushButton:checked{
	background-color:rgb(31, 78, 121);
	font-size:28px;
	border-radius:8px;
	color:white;
	border: 0px solid #1f4e79;    
};

</string>
       </property>
       <property name="text">
        <string notr="true">11</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	color: rgb(137, 137, 137);
	background-color: transparent;
	font-size:28px;
}</string>
       </property>
       <property name="text">
        <string notr="true">:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Calendar_btn_Second">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{ 
	background-color:rgb(255,255,255);
	font-size:28px; 
	border-radius:8px;
	color:rgb(31, 78, 121);
	border: 2px solid #1f4e79;    
}
QPushButton:hover
{
	background-color:rgb(31, 78, 121);
	font-size:28px;
	border-radius:8px;
	color:white;
	border: 0px solid #1f4e79;    
}

QPushButton:checked{
	background-color:rgb(31, 78, 121);
	font-size:28px;
	border-radius:8px;
	color:white;
	border: 0px solid #1f4e79;    
};

</string>
       </property>
       <property name="text">
        <string notr="true">11</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1,0,1">
     <item>
      <widget class="QPushButton" name="Calendar_btn_Cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{ 
	background-color:rgb(64,64,64);
	font-size:28px; 
	border-radius:8px;
	color:rgb(255,255,255);
}
QPushButton:pressed
{
	background-color:rgb(31, 78, 121);
	color:rgb(255,255,255);
	font-size:28px;
	border-radius:8px;
	/*border: 2px solid #1B72BF;    */
}
</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Calendar_btn_Now">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{ 
	background-color:rgb(64,64,64);
	font-size:28px; 
	border-radius:8px;
	color:rgb(255,255,255);
}
QPushButton:pressed
{
	background-color:rgb(31, 78, 121);
	color:rgb(255,255,255);
	font-size:28px;
	border-radius:8px;
	/*border: 2px solid #1B72BF;    */
}
</string>
       </property>
       <property name="text">
        <string>现在</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Calendar_btn_Ok">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{ 
	background-color:rgb(64,64,64);
	font-size:28px; 
	border-radius:8px;
	color:rgb(255,255,255);
}
QPushButton:pressed
{
	background-color:rgb(31, 78, 121);
	color:rgb(255,255,255);
	font-size:28px;
	border-radius:8px;
	/*border: 2px solid #1B72BF;    */
}
</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
